src/platform/site-wide/announcements/config/index.js

```diff
@@ -2,12 +2,13 @@ import DashboardIntro from '../components/DashboardIntro';
 import Profile360Intro from '../components/Profile360Intro';
 import PersonalizationBanner from '../components/PersonalizationBanner';
 import ClaimIncreaseBanner from '../components/ClaimIncreaseBanner';
+import isBrandConsolidationEnabled from '../../../brand-consolidation/feature-flag';
 
 const config = {
   announcements: [
     {
       name: 'dashboard-intro',
-      paths: /^(\/dashboard\/)$/,
+      paths: isBrandConsolidationEnabled() ? /^(\/my-va\/)$/ : /^(\/dashboard\/)$/,
```

Comment:
left this as is now- I don't know how to test this and doing a regex replace seemed like it would obfuscate the intent.